.github/workflows/ci.yml

```
@@ -83,6 +63,13 @@ jobs:
           - os: ubuntu-latest
             python-version: "3.10"
             extras: true
+          # Exclude pypy on windows due to significant performance issues
+          # running pytest
```

```suggestion
          # running pytest requires x min instead of y min on other platforms
```
it's a good idea to include numbers to have a reference.

---

Done